src/theme/colors.ts

```diff
@@ -2,138 +2,139 @@ const colors = {
   transparent: 'transparent',
   current: 'currentColor',
   inherit: 'inherit',
-  white: '#ffffff',
+  white: '#FFFFFF',
   black: '#000000',
 
-  'white-100': '#fefefe',
+  'white-100': '#FFFFFF',
+  'white-200': '#FEFEFE',
+  'white-300': '#F6F6F6',
 
-  'navyblue-100': '#86a3c3',
-  'navyblue-200': '#4f627c',
-  'navyblue-300': '#3e516b',
-  'navyblue-350': '#30415C',
-  'navyblue-400': '#23344e',
-  'navyblue-500': '#1d2a3f',
-  'navyblue-600': '#192231',
-  'navyblue-700': '#141c2a',
-  'navyblue-800': '#0f1623',
-
-  'gray-50': '#f6f6f6',
```

Comment:
I'm ok with the capitalization / alphabetical, but I don't think this PR should do changes on the color system itself. Can we undo the breaking changes to color tokens and keep this PR scoped only to button variants?

The button change is a small, low-risk change, but changing the color tokens will require migration steps in `panther-enterprise` and likely more discussion.

Reply:
I don't know if this will require a lot of migration steps. It will be a simple mass replacement from gray-50 to white-300 but i cannot see the white-300 in the Figma color pallet either so i don't know if this change is required or not.

Reply:
I have asked the design team ( Jim ) about white-*. And I will come back to you!

@benjamminj this task is about the colors and not the button ( Added because this PR looking a good place for this change - We could open another one too). 

The change on the format (Capital) was because it will be easier to copy-paste the color value from Figma to pounce. 

I think this PR doesn't have any breaking changes and the migration steps doesn’t require any big effort.
In any case if you aware of this we could close the PR. ( I made this PR based [on this thread](https://panther-labs.slack.com/archives/C02NVPZ1ZL4/p1638386009011000)! )

 [Created task ](https://app.asana.com/0/1200908948600021/1201451000624090/f) for the migration.

Reply:
If you're good to do the migration of tokens after we publish, I'm ok with that.

Agree with @kouknick, not sure about `white-300`, I only see `white-100` and `white-200` in the figma palette

Reply:
So after a discussion I had with Jim, the gray-50 is going to be replaced by white-200!
The migration step is just a simply replacement!